test/functional/inspec_exec_test.rb

```diff
@@ -1347,12 +1347,12 @@ def stderr
 
     it "should show enhanced_outcomes for controls with impact 0" do
       _(run_result.stdout).must_include "5 skipped"
-      _(run_result.stdout).must_include "3 controls not applicable"
+      _(run_result.stdout).must_include "4 controls not applicable"
```

Comment:
While going through the test I felt that we could have all enhanced outcomes and attestation tests in different files as we did for the streaming progress bar test. As this inspec exec test file is getting bigger and bigger. This is not something as a blocker for this PR just a suggestion.